docs/porting/tools/exporter.md

```diff
@@ -20,7 +20,7 @@ The export subsystem is organized as a group of common code and a group of IDE o
 The **common code** is contained in three files:
 
  * `tools/project.py` contains the entry point of the script to export projects from the Mbed tools to any of the supported IDEs or project structures.
- * `tools/export/__init__.py` contains an API used by the Arm Mbed Online Compiler and `project.py`. This file is responsible for boilerplate actions that every exporter must do, such as scanning for resources or collecting toolchain flags.
+ * `tools/export/__init__.py` contains an API used by `project.py`. This file is responsible for boilerplate actions that every exporter must do, such as scanning for resources or collecting toolchain flags.
```

Comment:
@saheerb Is this bit still technically correct?
See this page: https://os.mbed.com/docs/mbed-os/v6.15/porting/adding-exporters.html

Reply:
Exporters were deprecated. We should remove this documentation.

Reply:
https://os.mbed.com/blog/entry/Introducing-the-new-Mbed-Tools/ - not just exporters, entire tools were made read-only.

I've checked tools directory, exporters.md could be removed with this update (and one reference to it from tools.md file there).

Reply:
Ok. I removed the "Adding exporters" section https://os.mbed.com/docs/mbed-os/v6.15/porting/adding-exporters.html and updated the Porting tools introduction https://os.mbed.com/docs/mbed-os/v6.15/porting/porting-tools.html

Reply:
@0xc0170 There are many more references to "exporters". Should all these pages below be updated?

https://os.mbed.com/docs/mbed-os/v6.15/introduction/glossary.html
https://os.mbed.com/docs/mbed-os/v6.15/porting/porting-exporter-targets.html
https://os.mbed.com/docs/mbed-os/v6.15/program-setup/creating-and-using-a-bootloader.html
https://os.mbed.com/docs/mbed-os/v6.15/build-tools/create.html
https://os.mbed.com/docs/mbed-os/v6.15/build-tools/migration-guide.html
https://os.mbed.com/docs/mbed-os/v6.15/build-tools/about-the-exporters.html
https://os.mbed.com/docs/mbed-os/v6.15/build-tools/third-party-build-tools.html#about-the-exporters

Reply:
> https://os.mbed.com/docs/mbed-os/v6.15/build-tools/migration-guide.html

Reply:
@0xc0170 After talking to Joe, I'll roll back the changes I've made so far and add a deprecation note about exporters. So the content will stay as is but people will be warned. We want to publish the changes with the Online Compiler as soon as possible.